.github/workflows/upgrade-python-requirements.yml

```diff
@@ -39,7 +39,7 @@ jobs:
 
       - name: setup testeng-ci
         run: |
-          git clone https://github.com/edx/testeng-ci.git
+          git clone https://github.com/openedx/testeng-ci.git
```

Comment:
Note that this is a mistake - this repo has stayed in the edx org.

the correct solution here is this repo should be using the template defined in .github/workflow-templates. I've made this change here: https://github.com/openedx/RateXBlock/pull/48